### PR TITLE
[ISSUE #118] Add connect eventbridge 

### PR DESCRIPTION
**Pull request by zhaohai666**
#118

### CODE REVIEW

File: connectors/aliyun/rocketmq-connect-eventbridge/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+http://${runtime-ip}:${runtime-port}/connectors/${rocketmq-eventbridge-connector-name}/stop
+```
+
+## rocketmq-connect-eventbridge 参数说明
+* **eventbridge-sink-connector 参数说明**
+
+|         KEY            |  TYPE   | Must be filled | Description                      | Example
+|------------------------|---------|----------------|----------------------------------|--|
+|accessKeyId             | String  | YES            | 阿里云身份验证，在阿里云用户信息管理控制台获取                    | xxxx |
+|accessKeySecret         | String  | YES            | 阿里云身份验证，在阿里云用户信息管理控制台获取                     | xxx |
```
Comment (2011shenlin) [inline connectors/aliyun/rocketmq-connect-eventbridge/README.md:44]: Use the RAM role , not the ak/sk.

File: connectors/aliyun/rocketmq-connect-eventbridge/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+|accountEndpoint         | String  | YES            | 阿里云EventBridge官方接入点                     | xxxx |
+|eventId                 | String  | YES            | 事件ID | xxxx |
+|eventSource             | String  | YES            | 事件源 | xxxx |
+|eventType               | String | YES             | 事件类型                           | null |
```
Comment (2011shenlin) [inline connectors/aliyun/rocketmq-connect-eventbridge/README.md:48]: the attribute of event, extract from the connect record, not from the params of connector.

File: connectors/aliyun/rocketmq-connect-eventbridge/README.md (hunk 1 above)
Comment (zhaohai666) [reply connectors/aliyun/rocketmq-connect-eventbridge/README.md]: Modified to use RAM role

File: connectors/aliyun/rocketmq-connect-eventbridge/README.md (hunk 2 above)
Comment (zhaohai666) [reply connectors/aliyun/rocketmq-connect-eventbridge/README.md]: Properties of the modified event, extracted from the connection record.

File: connectors/aliyun/rocketmq-connect-eventbridge/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+|------------------------|---------|----------------|----------------------------------|--|
+|stsEndpoint             | String  | YES            | STS endpoint                       | xxxx|
+|accessKeyId             | String  | YES            | 阿里云授信账号的AK                    | xxxx |
+|accessKeySecret         | String  | YES            | 阿里云授信账号的SK                     | xxx |
+|roleArn                 | String  | YES            | 要扮演的RAM角色ARN。 该角色是可信实体为阿里云账号类型的RAM角色                     | xxxx |
+|roleSessionName         | String  | YES            | 角色会话名称。 该参数为用户自定义参数。通常设置为调用该API的用户身份                  | 用户名 |
+|eventTime               | String | YES             | 事件产生的时间                          | xxxx |
+|eventSubject            | String | NO              | 事件主题                          | xxxx |
+|aliyuneventbusname      | String | YES             | 接收事件的事件总线名称                          | xxxx |
+|connect-topicname       | String  | YES            | sink需要处理数据消息topic                     | xxxx |
```
[the remark this answers is not in the document]
Comment (zhaohai666) [reply connectors/aliyun/rocketmq-connect-eventbridge/README.md:50]: 这个是运行的时候会去校验是否配置了链接主题，如果没有配置会报错。

File: connectors/aliyun/rocketmq-connect-eventbridge/README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+## rocketmq-connect-eventbridge 参数说明
+* **eventbridge-sink-connector 参数说明**
+
+|         KEY            |  TYPE   | Must be filled | Description                      | Example
+|------------------------|---------|----------------|----------------------------------|--|
+|stsEndpoint             | String  | YES            | STS endpoint                       | xxxx|
+|accessKeyId             | String  | YES            | 阿里云授信账号的AK                    | xxxx |
+|accessKeySecret         | String  | YES            | 阿里云授信账号的SK                     | xxx |
+|roleArn                 | String  | YES            | 要扮演的RAM角色ARN。 该角色是可信实体为阿里云账号类型的RAM角色                     | xxxx |
```
[the remark this answers is not in the document]
Comment (zhaohai666) [reply connectors/aliyun/rocketmq-connect-eventbridge/README.md]: processed

File: connectors/aliyun/rocketmq-connect-eventbridge/README.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+|roleSessionName         | String  | YES            | 角色会话名称。 该参数为用户自定义参数。通常设置为调用该API的用户身份                  | 用户名 |
+|eventTime               | String | YES             | 事件产生的时间                          | xxxx |
+|eventSubject            | String | NO              | 事件主题                          | xxxx |
```
[the remark this answers is not in the document]
Comment (zhaohai666) [reply connectors/aliyun/rocketmq-connect-eventbridge/README.md]: processed

File: connectors/aliyun/rocketmq-connect-eventbridge/README.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+* **eventbridge-sink-connector 参数说明**
+
+|         KEY            |  TYPE   | Must be filled | Description                      | Example
+|------------------------|---------|----------------|----------------------------------|--|
+|stsEndpoint             | String  | YES            | STS endpoint                       | xxxx|
+|accessKeyId             | String  | YES            | 阿里云授信账号的AK                    | xxxx |
+|accessKeySecret         | String  | YES            | 阿里云授信账号的SK                     | xxx |
+|roleArn                 | String  | YES            | 要扮演的RAM角色ARN。 该角色是可信实体为阿里云账号类型的RAM角色                     | xxxx |
+|roleSessionName         | String  | YES            | 角色会话名称。 该参数为用户自定义参数。通常设置为调用该API的用户身份                  | 用户名 |
+|eventTime               | String | YES             | 事件产生的时间                          | xxxx |
```
[the remark this answers is not in the document]
Comment (zhaohai666) [reply connectors/aliyun/rocketmq-connect-eventbridge/README.md]: processed

File: connectors/aliyun/rocketmq-connect-eventbridge/src/main/java/org/apache/rocketmq/connect/eventbridge/sink/EventBridgeSinkConnector.java [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+    public Class<? extends Task> taskClass() {
+        return EventBridgeSinkTask.class;
+    }
+
+    @Override
+    public void validate(KeyValue config) {
+        if (StringUtils.isBlank(config.getString(EventBridgeConstant.ACCESS_KEY_ID))
+                || StringUtils.isBlank(config.getString(EventBridgeConstant.ACCESS_KEY_SECRET))
+                || StringUtils.isBlank(config.getString(EventBridgeConstant.ACCOUNT_ENDPOINT))
+                || StringUtils.isBlank(config.getString(EventBridgeConstant.ROLE_ARN))
```
Comment (2011shenlin) [inline connectors/aliyun/rocketmq-connect-eventbridge/src/main/java/org/apache/rocketmq/connect/eventbridge/sink/EventBridgeSinkConnector.java:74]: ROLE ARN \ROLE SESSION NAME\ STS is not necessary  when in ak mode directly.

Comment (zhaohai666) [reply connectors/aliyun/rocketmq-connect-eventbridge/src/main/java/org/apache/rocketmq/connect/eventbridge/sink/EventBridgeSinkConnector.java]: processed

File: connectors/aliyun/rocketmq-connect-eventbridge/src/main/java/org/apache/rocketmq/connect/eventbridge/sink/EventBridgeSinkConnector.java [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+            throw new RuntimeException("EventBridge required parameter is null !");
+        }
+    }
+
+    @Override
+    public void init(KeyValue config) {
+        accessKeyId = config.getString(EventBridgeConstant.ACCESS_KEY_ID);
+        accessKeySecret = config.getString(EventBridgeConstant.ACCESS_KEY_SECRET);
+        roleArn = config.getString(EventBridgeConstant.ROLE_ARN);
+        roleSessionName = config.getString(EventBridgeConstant.ROLE_SESSION_NAME);
```
[the remark this answers is not in the document]
Comment (zhaohai666) [reply connectors/aliyun/rocketmq-connect-eventbridge/src/main/java/org/apache/rocketmq/connect/eventbridge/sink/EventBridgeSinkConnector.java:90]: processed

**Review by 2011shenlin — APPROVED** [review]

**Pull request merged**